styles/user-create-form.css

```diff
@@ -0,0 +1,33 @@
+.container-user-form{
+    background:url();
```

Comment:
do we need this?
we dont have an url for this

Reply:
we don't need this , it was delete